nomt/src/bitbox/mod.rs

```diff
@@ -139,6 +139,14 @@ impl DB {
             ht_pages.push((pn, buf));
         }
 
+        if cfg!(debug_assertions) {
+            // Make sure that there are no duplicate pages.
```

Comment:
I am not sure this will pass, actually...

note that our `BucketAllocator` currently gives out buckets freed by deletes within the same transaction.
We can either change that behavior _or_ find a way to skip those deletions in writeout.

Reply:
Yes...

Reply:
https://github.com/thrumdev/nomt/pull/347